pkg/openthread/Makefile.include

```diff
@@ -2,7 +2,9 @@ OPENTHREAD_DIR = $(RIOTBASE)/pkg/openthread
 
 INCLUDES += -I$(OPENTHREAD_DIR)/include \
             -I$(BINDIRBASE)/pkg/$(BOARD)/openthread/output/include \
+            -I$(BINDIRBASE)/pkg/$(BOARD)/openthread/include \
             -I$(BINDIRBASE)/pkg/$(BOARD)/openthread/include/openthread \
+            -I$(BINDIRBASE)/pkg/$(BOARD)/openthread/include/openthread/platform \
```

Comment:
Why are this one and the previous one required? The one that are subdirectories from `include`.
It compiles without them and the C files use include with `openthread/header.h` and `openthread/platform/header.h`.

Also, when comparing the diff between the build directories with and without the lines, the generated `.d` files are the same so suggest that indeed it did not change any include path.

Reply:
Also, for checking right now, I think there is not reason to depend on `openthread/output/include` directory.
(except that the `sed` on `types.h` should be done with a patch in the main `include` directory).

It would also be better because having other modules depending on openthread being compiled before them to have the `output` directory is not something entirely guaranteed. Normally only the `prepare` has been done before.
For packages there is still a rule to build them before other modules but it is more a side-effect than a real ensured property. Also another package that would depend on this header would have problem as it may not exist.